src/CMakeLists.txt

```diff
@@ -188,6 +188,15 @@ if(USE_GOLD_LINKER)
                  -fuse-ld=gold)
 endif()
 
+# Temporary staging; the various swift projects that depend on libdispatch
+# all expect libdispatch.so to be in src/.libs/libdispatch.so
+# So for now, make a copy so we don't have to do a coordinated commit across
+# all the swift projects to change this assumption.
+add_custom_command(TARGET dispatch POST_BUILD
+                   COMMAND cmake -E make_directory .libs
+                   COMMAND cmake -E copy $<TARGET_FILE:dispatch> .libs
+                   COMMENT "Copying libdispatch to .libs")
```

Comment:
Ah, I guess I was mixing up some other behaviour then.  If it is only run once, then it is what we want.